src/components/BlockingViews/BlockingView.js

```diff
@@ -51,7 +54,12 @@ const BlockingView = props => (
         {props.shouldShowBackHomeLink
             ? (
                 <TextLink
-                    onPress={() => Navigation.navigate(ROUTES.REPORT)}
+                    onPress={() => {
+                        if (props.isSmallScreenWidth) {
```

Comment:
@allroundexperts Is it necessary to check if we are on smallscreen? I think we can just `Navigation.dismissModal(true)` onpress.